### ISSUE TITLE
Better egg not working for all players.

### ISSUE THREAD

**Issue by DravenXD** [nexor-source/ControllableExplosiveEgg#1]
The better egg mode only works for me, it does not work for any of my friends. They all turned on better egg in the config.ini but it still doesn't work for them, it only works for me.
より良いエッグモードは私にのみ機能し、私の友人には機能しません。彼らは皆、config.ini でより良い卵を有効にしましたが、それでも彼らにとっては機能せず、私にとってのみ機能しました。translate.google.com

Edit: 
GPT translation: 私だけが「ベターエッグモード」が機能するんです。友達たち全員がconfig.iniでベターエッグを有効にしましたが、彼らにはまだ機能していません。

**Comment by nexor-source** [comment]
> The better egg mode only works for me, it does not work for any of my friends. They all turned on better egg in the config.ini but it still doesn't work for them, it only works for me. より良いエッグモードは私にのみ機能し、私の友人には機能しません。彼らは皆、config.ini でより良い卵を有効にしましたが、それでも彼らにとっては機能せず、私にとってのみ機能しました。translate.google.com
> 
> Edit: GPT translation: 私だけが「ベターエッグモード」が機能するんです。友達たち全員がconfig.iniでベターエッグを有効にしましたが、彼らにはまだ機能していません。

@DravenXD 
Yes, currently better-egg mode is only useful when acting as a host (server). I don't know how to make this work for everyone.
是的，目前better-egg mode只能在作为房主时有用。我暂时并不知道怎么让这个功能对所有人都生效

**Comment by DravenXD** [comment]
I understand, thank you for your mod. I hope maybe one day we can get this working for everyone in the server.
GPT: 了解しました。モッドをありがとうございます。いつかこのモッドがサーバーのみんなにとって動作するようになることを願っています。
